CHANGELOG.md

```diff
@@ -13,6 +13,18 @@ This project does NOT adhere to [Semantic Versioning](https://semver.org/spec/v2
   * `import { editingDataService } from '@sitecore-jss/sitecore-jss-nextjs/editing';`
   * `import { EditingRenderMiddleware } from '@sitecore-jss/sitecore-jss-nextjs/editing';`
 
+* `[sitecore-jss-angular]` jss-angular package and sample has been updated to version 14. This means several changes:
+  * JSS Angular apps are now using Ivy
+  * IE11 no longer supported by JSS Angular
+  * _sitecore-jss-angular_ package does not output UMD package anymore - only ESM. Because of this constants are now a separate JSON file in sample app under scripts, and are not imported from the _sitecore-jss-angular_ package
+  * _componentFactory_ is no longer present in ComponentFactoryResult interface, due to _createComponent_ changes and deprecations introduced in Angular 13
+  * More details on changes in Angular can be found in the below links:
+  https://blog.angular.io/angular-v13-is-now-available-cce66f7bc296
+  https://angular.io/guide/deprecations
+  https://update.angular.io/?l=3&v=11.0-14.0 
+
+* `[sitecore-jss-angular]` Due to the Angular version upgrade and the change in _sitecore-jss-angular_ package output format  _sitecore-jss_ exports are not available in angular app scripts (src/scripts) via '@sitecore-jss/sitecore-jss-angular'. Please use '@sitecore-jss/sitecore-jss-angular/base-pkg-cjs' import instead. Check bootstrap.ts scripts as for a usage example.  
```

Comment:
I would suggest to use name: `@sitecore-jss/sitecore-jss-angular/cjs`, it's possible to put here whatever you will decide, not only `sitecore-jss` related items

Reply:
```suggestion
* `[sitecore-jss-angular]` Due to the Angular version upgrade and the change in _sitecore-jss-angular_ package output format  _sitecore-jss_ exports are not available in angular app scripts (src/scripts) via '@sitecore-jss/sitecore-jss-angular'. Please use '@sitecore-jss/sitecore-jss-angular/cjs' import instead. Check bootstrap.ts scripts as for a usage example.  
```